helpers/config.py

```diff
@@ -761,11 +773,12 @@ def __questions_aws(self):
         Asks if user wants to see AWS option
         and asks for credentials if needed.
         """
-        self.__dict['use_aws'] = CLI.yes_no_question(
-            'Do you want to use AWS S3 storage?',
-            default=self.__dict['use_aws']
-        )
-        if self.__dict['use_aws'] is True:
+        if not self.__dict['use_aws']:
```

Comment:
The idea of the setup is to rebuild all the configuration each time. I understand that the chances are small that one environment move from `AWS` to local storage but users should be able to make the change. With this new condition, it would not be possible :-( 
Would you mind to revert this  change please?